app/models/organization.js

```diff
@@ -13,4 +13,6 @@ export default class OrganizationModel extends Model {
   @hasMany('press-release') pressReleases;
   @hasMany('collaboration-activity', { inverse: 'collaborators' }) collaborations;
   @hasMany('collaboration-activity', { inverse: 'editor' }) editorCollaborations;
+  @hasMany('approval-activity', { inverse: 'collaborator' }) approvalActivities;
```

Comment:
Aangezien er maar 1 relatie is vanuit `organization` naar `approval-activity` moet je de inverse niet specifiëren